packages/netlify-cms-core/src/components/Editor/Editor.js

```diff
@@ -133,20 +133,20 @@ class Editor extends React.Component {
     }
   }
 
-  componentWillReceiveProps(nextProps) {
+  componentDidUpdate(prevProps) {
     /**
      * If the old slug is empty and the new slug is not, a new entry was just
      * saved, and we need to update navigation to the correct url using the
      * slug.
      */
```

Comment:
[Side effects on props change should be in `componentDidUpdate`](https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html#side-effects-on-props-change)